package.json

```diff
@@ -15,12 +15,14 @@
     "lint": "standard --fix"
   },
   "dependencies": {
-    "@cmfcmf/docusaurus-search-local": "^0.10.0",
-    "@docusaurus/core": "^2.0.0-beta.20",
-    "@docusaurus/plugin-google-gtag": "^2.0.0-beta.20",
-    "@docusaurus/plugin-sitemap": "^2.0.0-beta.20",
-    "@docusaurus/preset-classic": "^2.0.0-beta.20",
-    "@docusaurus/theme-search-algolia": "^2.0.0-beta.20",
+    "@cmfcmf/docusaurus-search-local": "^0.11.0",
```

Comment:
Why are we taking on alpha dependencies?

Reply:
I updated the package version with a stable one